model/user.js

```diff
@@ -0,0 +1,42 @@
+/*
+https://scotch.io/bar-talk/using-mongoosejs-in-node-js-and-mongodb-applications
+*/
+
+const mongoose = require('mongoose')
+
+// create a schema
+const userSchema = new mongoose.Schema({
+  name: String,
+  fbId: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  name: String,
+  activeGame: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Game'
+  }, // if a logout occurs can fetch gameID
+  admin: Boolean,
+  created_at: {
+    type: Date,
+    default: Date.now
+  },
+  updated_at: Date
+  /* Things we don't strictly need atm
```

Comment:
should we just remove this for now?

Reply:
@DarkPurple141

Reply:
@zainafzal08 remove cos the update didn't seem to work for you. Bad Zain. Bad Zain. Sadly a pain I suffered myself yesterday, the data object returned by update is the old data object. Thus your assertion was failing.

Reply:
The way to avoid this according the docs is add the option { new : true }